data/assets/util/webgui.asset

```diff
@@ -4,7 +4,7 @@ local guiCustomization = asset.require("customization/gui")
 
 
 -- Select which commit hashes to use for the frontend and backend
-local frontendHash = "1e64af3b19764af15ad844dec543b1fffec99fe9"
+local frontendHash = "580386a78b0fda3808c4c00c120ab18aace7dc22"
```

Comment:
What do we want to do with the route further down in this file?  That one does not seem up to date with the new UI structure

E.g. now the URL that is loaded is still http://localhost:4670/frontend/#/onscreen, which works, but is the same as just loading http://localhost:4670/frontend

Reply:
It seems we set this function in the base_blank asset. Seems to me we could just remove this function and just default to frontend?

```
asset.onInitialize(function()
  webGui.setCefRoute("onscreen")
  openspace.setPropertyValueSingle("RenderEngine.VerticalLogOffset", 0.1)
  openspace.setPropertyValueSingle("Dashboard.StartPositionOffset", { 15.0,-49.0 })
  openspace.setPropertyValueSingle("RenderEngine.ShowCamera", false)
end)

```